internal/builders/maven/action.yml

```diff
@@ -47,6 +47,10 @@ outputs:
       Users should verify the download against this digest to prevent tampering.
     value: ${{ steps.upload-target.outputs.sha256 }}
 
+  package-download-name:
```

Comment:
nit: let's call it `target-download-name` for consistency?

Reply:
Should have fixed all names.